src/vulkan-renderer/application.cpp

```diff
@@ -70,6 +70,19 @@ void Application::load_toml_configuration_file(const std::string &file_name) {
     const auto &configuration_title = toml::find<std::string>(renderer_configuration, "title");
     spdlog::debug("Title: '{}'", configuration_title);
 
+    using WindowMode = ::inexor::vulkan_renderer::wrapper::Window::Mode;
+    const auto &wmodestr = toml::find<std::string>(renderer_configuration, "application", "window", "mode");
+    if (!wmodestr.compare("windowed")) {
+        m_window_mode = WindowMode::WINDOWED;
+    } else if (!wmodestr.compare("windowedfullscreen")) {
```

Comment:
```suggestion
    } else if (!wmodestr.compare("windowed_fullscreen")) {
```

might be a better solution

Reply:
Wouldn't @yeetari propose to just use `==` operator instead?

Reply:
Was thinking about it too, not sure how it looks in the rest of the source.

Reply:
Yes I think `==` should be used here instead. It's more readable imo.

Reply:
I have a commit that does this but I forgot to push it! Will do it later today.